src/charts/trends/trends-chart.html

```diff
@@ -19,7 +19,7 @@
   </div>
   <div ng-switch-when="compact" class="trend-card-compact-pf">
     <div class="row trend-row">
-      <div class="col-sm-4 col-md-4">
+      <div class="col-sm-2 col-md-2">
```

Comment:
Now would be a good time to lose the useless col-md values on these columns since they are the same as the col-sm value. Would clean it up and one less selector in the less file above.